src/content/docs/public-pool/selfhost.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Self-Hosting Public Pool
+---
+
+1. 💻 [Windows](#-windows) - TODO
+2. 🍏 [MacOS](#-macos) - TODO
+3. 🐧 [Linux](#-linux)
```

Comment:
oh also highkey this isnt really necessary for this page, buut i dont think branching in each section would be readable

it would only be the deps....